source/fundamentals/crud/read-operations/limit.txt

```diff
@@ -40,66 +52,66 @@ To limit the number of documents returned from a query, pass the
 number of documents you want returned to the ``SetLimit()`` method of
 the read operation's options.
 
-Specify the options as the last parameter to the following read
-operation methods:
+The following read operations take an options object as a parameter:
 
 - ``Find()``
 - ``CountDocuments()``
 - ``gridfs.Bucket.Find()``
 
 If the limit is ``0`` or exceeds the number of matched
-documents, the method returns all the documents.  If the limit is a
-negative number, the method behaves as if the limit was the absolute
-value of the negative number and closes the cursor after retrieving
+documents, the method returns all the documents. If the limit is a
+negative number, the method take the absolute value of the negative
+number as the limit and closes the cursor after retrieving
 documents.
 
 Example
 ~~~~~~~
 
-The following example shows how to return two documents:
+The following example shows how to return two documents that have an
+``enrollment`` field value greater than 20:
 
 .. io-code-block::
    :copyable: true
 
    .. input::
       :language: go
 
-      filter := bson.D{}
+      filter := bson.D{{"enrollment", bson.D{{"$gt", 20}}}}
       opts := options.Find().SetLimit(2)
-
+      
       cursor, err := coll.Find(context.TODO(), filter, opts)
-
-      var results []bson.D
+      
+      var results []Course
       if err = cursor.All(context.TODO(), &results); err != nil {
-         panic(err)
+          panic(err)
       }
       for _, result := range results {
-         fmt.Println(result)
+          res, _ := json.Marshal(result)
+          fmt.Println(string(res))
       }
 
    .. output::
       :language: none
       :visible: false
 
-      [{_id ObjectID("...")} {type Masala} {rating 10}]
-      [{_id ObjectID("...")} {type Assam} {rating 5}]
+      {"Title":"Concepts in Topology","Enrollment":35}
+      {"Title":"Ancient Greece","Enrollment":100}
 
 Multiple Options
 ----------------
 
 If you configure other options alongside the ``SetLimit()`` method,
-the driver performs the limit last regardless of the order you list
-the options.
+the driver performs the limit last regardless of the order in which you set
+the other options.
```

Comment:
Suggestion:

For clarity, I think it would be smoother to remove the conditional phrase at the beginning. For example:

"The driver performs the limit option last regardless of the order of options you set in the ``SetLimit()`` method."